test/edu-benefits/1990e/config/sponsorVeteran.unit.spec.jsx

```diff
@@ -60,8 +60,7 @@ describe('Edu 1990e sponsorVeteran', () => {
         value: 'Veteran'
       }
     });
-
-    ReactTestUtils.Simulate.change(find('#root_veteranSocialSecurityNumber'), {
+    ReactTestUtils.Simulate.change(find('#root_view:veteranId_veteranSocialSecurityNumber'), {
```

Comment:
If I recall correctly, it's the `:` in the id. To get around this, we've had to do something like:
```js
Array.from(find('input')).find((input) => input.id === 'root_view:veteranId_veteranSocialSecurityNumber')
```